apex/transformer/utils.py

```diff
@@ -8,6 +8,7 @@
 # The following 4 lines are for backward comparability with
 # older PyTorch.
 if "all_gather_into_tensor" not in dir(torch.distributed):
+    assert torch.distributed.is_available(), "PyTorch Distributed is Not available or Disabled."
```

Comment:
as we can technically ignore `assert` ([ref](https://docs.python.org/3/using/cmdline.html#cmdoption-O))
```suggestion
    if not torch.distributed.is_available():
        raise RuntimeError("PyTorch Distributed is Not available or Disabled.")
```